<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.0 (Python 3.9.0 on darwin)" generated="20230409 21:00:02.517" rpa="false" schemaversion="3">
<suite id="s1" name="Feature" source="/Users/refqihussein/Documents/tixid/test/feature/feature.robot">
<test id="s1-t1" name=" Tests Open App" line="9">
<kw name="Start Test" library="hook" type="SETUP">
<kw name="Set Appium Timeout" library="AppiumLibrary">
<arg>${TIMEOUT_MAX}</arg>
<doc>Sets the timeout in seconds used by various keywords.</doc>
<status status="PASS" starttime="20230409 21:00:02.852" endtime="20230409 21:00:02.853"/>
</kw>
<kw name="Open Application" library="AppiumLibrary">
<arg>${APPIUM_SERVER}</arg>
<arg>app=${APP_PATH}</arg>
<arg>deviceName=${DEVICE_NAME}</arg>
<arg>udid=${UDID}</arg>
<arg>automationName=${AUTOMATION_NAME}</arg>
<arg>platformName=${PLATFORM_NAME}</arg>
<arg>platformVersion=${PLATFORM_VERSION}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/server-args.md
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20230409 21:00:02.897" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20230409 21:00:02.896" endtime="20230409 21:00:02.907"/>
</kw>
<msg timestamp="20230409 21:00:02.912" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20230409 21:00:02.915" level="FAIL">WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The application at 'app/android/id.tix.android_2023-02-19.apk' does not exist or is not accessible
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'app/android/id.tix.android_2023-02-19.apk' does not exist or is not accessible
    at getResponseForW3CError (/usr/local/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
    at asyncHandler (/usr/local/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:380:37)</msg>
<status status="FAIL" starttime="20230409 21:00:02.854" endtime="20230409 21:00:02.916"/>
</kw>
<status status="FAIL" starttime="20230409 21:00:02.851" endtime="20230409 21:00:02.917"/>
</kw>
<kw name="Finish Test" library="hook" type="TEARDOWN">
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20230409 21:00:02.922" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20230409 21:00:02.921" endtime="20230409 21:00:02.923"/>
</kw>
<msg timestamp="20230409 21:00:02.929" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20230409 21:00:02.931" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20230409 21:00:02.920" endtime="20230409 21:00:02.933"/>
</kw>
<kw name="Close Application" library="AppiumLibrary">
<doc>Closes the current application and also close webdriver session.</doc>
<kw name="Capture Page Screenshot" library="AppiumLibrary">
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<msg timestamp="20230409 21:00:02.936" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20230409 21:00:02.935" endtime="20230409 21:00:02.940"/>
</kw>
<msg timestamp="20230409 21:00:02.945" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20230409 21:00:02.947" level="FAIL">No application is open</msg>
<status status="FAIL" starttime="20230409 21:00:02.933" endtime="20230409 21:00:02.947"/>
</kw>
<status status="FAIL" starttime="20230409 21:00:02.918" endtime="20230409 21:00:02.948">Several failures occurred:

1) No application is open

2) No application is open</status>
</kw>
<status status="FAIL" starttime="20230409 21:00:02.849" endtime="20230409 21:00:02.949">Setup failed:
WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The application at 'app/android/id.tix.android_2023-02-19.apk' does not exist or is not accessible
Stacktrace:
UnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'app/android/id.tix.android_2023-02-19.apk' does not exist or is not accessible
    at getResponseForW3CError (/usr/local/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/errors.js:804:9)
    at asyncHandler (/usr/local/lib/node_modules/appium/node_modules/appium-base-driver/lib/protocol/protocol.js:380:37)

Also teardown failed:
Several failures occurred:

1) No application is open

2) No application is open</status>
</test>
<doc>Example Feature Tests</doc>
<status status="FAIL" starttime="20230409 21:00:02.528" endtime="20230409 21:00:02.954"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Feature">Feature</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230409 21:00:02.912" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20230409 21:00:02.929" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg timestamp="20230409 21:00:02.945" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
